packages/devtools_app/lib/src/screens/memory/panes/diff/controller/sampler.dart

```diff
@@ -54,13 +54,15 @@ class HeapClassSampler extends ClassSampler {
         isolateId,
         theClass.id!,
       );
-    } catch (error, trace) {
-      _outputError(error, trace);
+    } catch (error) {
+      _outputError(error);
       return null;
     }
   }
 
-  void _outputError(Object error, StackTrace trace) {
+  void _outputError(
+    Object error,
```

Comment:
I think this change was a false positive. Line 66 should be `$trace` instead of `trace`.